lib/queue.js

```diff
@@ -884,6 +884,11 @@ Queue.prototype.retryJob = function(job) {
 };
 
 Queue.prototype.toKey = function(queueType){
+  return [this.keyPrefix ? '' : 'bull', this.name, queueType].join(':');
+};
+
+Queue.prototype.toPrefixedKey = function(queueType){
+  if (!this.keyPrefix) return this.toKey(queueType);
   return [this.keyPrefix, this.name, queueType].join(':');
 };
 
```

Comment:
Is there any difference between toKey and toPrefixedKey? Looks like they would return the same thing. Furthermore, what is the change actually doing? it also seems the old toKey worked exactly as the new one. I may be missing something subtle here...